### PR TITLE
[CSL-1775] Finish moving Txp into txp/

### PR DESCRIPTION
**Pull request by neongreen**

### CODE REVIEW

**Comment by neongreen** [comment]
Appveyor is broken :(

> ```
> git checkout -qf FETCH_HEAD
> Restoring build cache
> Error downloading cache item: Remote server returned 500: 
>   The process cannot access the file 'D:\cache-data\build-cache-53120
>     \project-265590\da39a3ee5e6b4b0d3255bfef95601890afd80709
>     \82eb3ad6bfe5a56e0e930991a581f6e732a27fea.zip.001\data'
>   because it is being used by another process.
> ```

File: lib/cardano-sl.cabal [hunk 1]
```diff
@@ -60,6 +59,8 @@ library
                         Pos.Behavior
 
                         -- Something
+                        Pos.Types
```
Comment (gromakovsky) [inline lib/cardano-sl.cabal]: Why did you move it here? 🤔 
They were alphabetically sorted (or very close to be sorted)

File: txp/Pos/Txp.hs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+-- | Txp system reexports.
+
+{-# OPTIONS_GHC -F -pgmF autoexporter #-}
+
+import           Pos.Arbitrary.Txp         ()
+import           Pos.Arbitrary.Txp.Network ()
```
Comment (gromakovsky) [inline txp/Pos/Txp.hs]: `Pos.Arbitrary.Txp.Network` imports `Pos.Arbitrary.Txp`, so the import of `Pos.Arbitrary.Txp` is likely redundant. On the other hand, it's kinda weird that this dependency is not the other way around 😕 
Anyway, it's not important. I'd like `Pos.Arbitrary.Txp` to import `Pos.Arbitrary.Txp.Network`, but not too much.

File: lib/cardano-sl.cabal (hunk 1 above)
Comment (neongreen) [reply lib/cardano-sl.cabal]: Because `-- Txp` was a wrong place for it

File: txp/Pos/Txp.hs (hunk 2 above)
Comment (neongreen) [reply txp/Pos/Txp.hs]: > I'd like Pos.Arbitrary.Txp to import Pos.Arbitrary.Txp.Network

me too, but I like the green checks so I might fix it in a different PR
